examples/tests/valid/api.w

```diff
@@ -0,0 +1,23 @@
+bring cloud;
+
+// let api = new cloud.Api();
+
+// let counter = new cloud.Counter();
+
+// let handler = inflight (request: cloud.ApiRequest): cloud.ApiResponse => {
+
+//   let count = counter.inc();
+
+//   let resp = cloud.ApiResponse {
+//     body: {
+//       count: count,
+//     },
+//     status: 200,
+//     headers: {
+//       "Content-Type": "text/plain",
+//     },
+//   };
+//   return resp;
+// };
+
+// api.get("/hello/world", handler);
```

Comment:
Can we uncomment this?

Reply:
Unless you have way to skip a hangar test for this file not until we have a simulator implementation.

Reply:
Ah I see... maybe just add a comment at the top of the file, or we can remove this test for now